src/createServerClient.ts

```diff
@@ -180,7 +173,7 @@ export function createServerClient<
     },
   );
 
-  client.auth.onAuthStateChange(async (event) => {
+  client.auth.onAuthStateChange(async (event: AuthEvent) => {
```

Comment:
Hmm this type can be implicitly inferred.

Reply:
Yes it is correct, but in this case TypeScript forces us to assign a type. I just assigned it appropriately by taking it from the supabase-js library